hoptimator-k8s/src/main/java/com/linkedin/hoptimator/k8s/K8sJobDeployer.java

```diff
@@ -42,7 +42,6 @@ public List<String> specify() throws SQLException {
         .with("schema", job.sink().schema())
         .with("table", job.sink().table())
         .with("sql", sql.apply(SqlDialect.ANSI))
-        .with("flinksql", sql.apply(SqlDialect.FLINK))
```

Comment:
I think we still need `{{flinksql}}`, and we should use that in our flink templates. But maybe it just maps to the same thing as `{{sql}}` for now.

Keep in mind Flink is unique wrt it's DDL. We need to know we are generating Flink DDL, even if the queries are just ANSI. If/when we want to target another engine (e.g. spark) we will need something like `{{sparksql}}` that uses different DDL, even if the dialect for queries is still just ANSI.

Reply:
OTOH, we have been talking about dropping DDL generation, so maybe it doesn't matter longer term what we do here.

Reply:
Fair point, I added back flinksql but just have `SqlDialect.FLINK` & `SqlDialect.ANSI` using the ANSI dialect